apps/backend-sls/test/integration.test.ts

```diff
@@ -1,13 +1,241 @@
+import { PrismaClient } from '@klicker-uzh/prisma'
```

Comment:
Created an initial integration test that logs in with the seeded user and creates a session, starts it, and activates a block. Interacts with the database and redis cache, though much is still missing.